### PR TITLE
Enable onnx conda package for Windows

### PR DESCRIPTION
**Pull request by jcwchen**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

Get ideas from https://github.com/conda-forge/onnx-feedstock/pull/45. Thank you @clouds56

### CODE REVIEW

**Comment by jcwchen** [comment]
@conda-forge-admin, please rerender

**Comment by jcwchen** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,18 +10,27 @@ source:
...
-  number: 3
+  number: 4
   script:
-    - export ONNX_ML=1  # [unix]
     - set "ONNX_ML=1"  # [win]
+    - set CONDA_PREFIX=%LIBRARY_PREFIX%  # [win]
+    - set CMAKE_GENERATOR=Ninja  # [win]
+    - set CMAKE_BUILD_TYPE=Release  # [win]
+    - set "CMAKE_ARGS=-DONNX_USE_PROTOBUF_SHARED_LIBS=OFF -DProtobuf_USE_STATIC_LIBS=ON"  # [win]
+    - set "PYTHON_EXECUTAsBLE=%PYTHON%"  # [win]
```
Comment (clouds56) [inline recipe/meta.yaml:20]:
```suggestion
    - set "PYTHON_EXECUTABLE=%PYTHON%"  # [win]
```

**Comment by marcelotrevisani** [comment]
lets separate it in two files, the ``build.sh`` for linux and osx and the ``bld.bat`` for windows
It is a bit confusing now we added more commands

**Comment by marcelotrevisani** [comment]
@conda-forge-admin , please rerender

**Comment by marcelotrevisani** [comment]
@jcwchen 
It seems to be fine now, would you like to integrate it as it is? or do you want to do something more?

**Review by marcelotrevisani — APPROVED** [review]

**Comment by jcwchen** [comment]
@marcelotrevisani Looking good. Thank you so much for the improvement!

**Pull request merged**